src/models/Product.ts

```diff
@@ -33,7 +33,6 @@ class Product
   public price!: number | null;
   public image!: string | null;
   public stallId!: number;
-
```

Comment:
I left a line in the other models to differentiate between `readonly` and the normal variables.

Reply:
Added